pkg/config/resource.go

```diff
@@ -312,12 +324,12 @@ type ListOperationConfig struct {
 	MatchFields []string `json:"match_fields"`
 }
 
-// UpdateOperationConfig contains instructions for the code generator to handle
-// Update operations for service APIs that have resources that have
-// difficult-to-standardize update operations.
-type UpdateOperationConfig struct {
```

Comment:
Since `omitUnchangedFields` doesn't apply to read operations, we can create a new struct called `ReadOperationConfig`

Reply:
done